app/models/blog.rb

```diff
@@ -9,9 +9,7 @@ class Blog < ApplicationRecord
 
   scope :published, -> { where('secret = FALSE') }
 
-  scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
-  }
+  scope :search, ->(term) { where('title LIKE ? OR content LIKE ?', "%#{term}%", "%#{term}%") }
```

Comment:
```suggestion
  scope :search, ->(term) { where('title LIKE :term OR content LIKE :term', ...) }
```

のように同じパラメータ名を渡すとコードがDRYになります。

Reply:
`scope :search, ->(term) { where('title LIKE :term OR content LIKE :term', term: "%#{term}%") }`としました。